src/vue/components/inflections-table-wide.vue

```diff
@@ -81,7 +81,7 @@
 
     <div
         class="alpheios-inflections__not-impl-msg"
-        v-show="!state.collapsed && !isAvailable"
+        v-if="!state.collapsed && !isAvailable"
```

Comment:
I have changed to v-if because the table could be implemented or not.
It won't change during workflow, so there is no sense to render it if it is implemented.